aten/src/ATen/native/vulkan/ops/Arithmetic.cpp

```diff
@@ -322,6 +322,13 @@ Tensor add_tensor(
     const Tensor& self_arg,
     const Tensor& other_arg,
     const Scalar& alpha) {
+  if (other_arg.sizes().size() == 0) {
```

Comment:
Do we have tests in case other_arg.sizes().size() is 0? If not, recommend you add new ones in a separate diff.